src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,11 +1,19 @@
 package core.mate.academy.model;
 
+import core.mate.academy.Color;
+import java.util.Random;
+
 /**
  * Add some custom fields that could be only in Bulldozer
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private static final Random RANDOM = new Random();
```

Comment:
```suggestion
    private Random random = new Random();
```

why it is constant? Why do we need Random in Bulldozer class? Bulldozer it's a model, lets take all it's parameters in constructor as incoming, not generate them there (same for other classes)

Reply:
добре, я перероблю. Але я зашив логіку в класс і він сам вирішує які генерувати назви, що досить природньо. В попередньому коментарі було використано слово Випадково, а не в ручну - тому було зроблено так як зроблено

Reply:
і все одно не зрозуміло чому моя реалізація не підходить і в чому моя помилка як програміста

Reply:
Є загальноприйнята структура Java коду. Є класи модельки такі як Bulldozer, які описують якусь сутність і зберігають її стан або специфічну для неї поведінку. Є класи сервіси, в яких описується основна бізнес логіка твоєї программи і які оперують модельками. Логіка по створенню рандомних параметрів не відноситься до модельки, вона тобі потрібна тобі в конкретному класі BulldozerProducer. Для інших класів, які оперують цією моделькою вона скоріш за все не потрібна, тому не бачу причин імплементовувати її саме в классі Bulldozer